### PR TITLE
Gabriela Argüelles: Assignment 4 (RDFLib)

### PR DESCRIPTION
**Pull request by gabyarte**

### CODE REVIEW

File: Assignment4/gabyarte-e22a357/Task07.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+# %% [markdown]
+# **TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**
+# 
+
+# %%
+q2 = prepareQuery('''
+        SELECT ?individual
+        WHERE {{
+            ?subclass rdfs:subClassOf ns:Person .
```
Comment (dgarijo) [inline Assignment4/gabyarte-e22a357/Task07.py:53]: use * to get all subclasses of person.

File: Assignment4/gabyarte-e22a357/Task07.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+q3 = prepareQuery('''
+        SELECT ?individual ?property ?value
+        WHERE {{
+            ?subclass rdfs:subClassOf ns:Person .
+            ?individual a ?subclass .
+            ?individual ?property ?value
+        }
+        UNION {
+            ?individual a ns:Person .
+            ?individual ?property ?value
```
Comment (dgarijo) [inline Assignment4/gabyarte-e22a357/Task07.py:89]: same as above

File: Assignment4/gabyarte-e22a357/Task06.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+# Visualize the results
+for s, p, o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+
+# %% [markdown]
+# **TASK 6.5: Add UPM as the university where John Smith works**
+
+# %%
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, VCARD.Work, ns.UPM))
```
[the remark this answers is not in the document]
Comment (gabyarte) [reply Assignment4/gabyarte-e22a357/Task06.py]: Updated on PR

File: Assignment4/gabyarte-e22a357/Task07.py (hunk 1 above)
Comment (gabyarte) [reply Assignment4/gabyarte-e22a357/Task07.py]: Updated on PR

File: Assignment4/gabyarte-e22a357/Task07.py (hunk 2 above)
Comment (gabyarte) [reply Assignment4/gabyarte-e22a357/Task07.py]: Updated on PR

**Pull request merged**